### PR TITLE
Translate menu from xlf (not _lang table)

### PR DESCRIPTION
**Pull request by julienbourdeau**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Enable menu to be translated through Crowdin just like anything else! This may not look very pretty but it guarantees more backward compatibility since database is not changed
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/STAB-17
| How to test?  |

### TODO

- [x] Load translation from XLF so they can be handled on Crowdin, like everything else
- [x] Get rid of xml files in `install` folder
- [ ] Do the same for modules (trans must call something like `getModuleTranslation()`

### CODE REVIEW

File: classes/Tab.php [hunk 1]
```diff
@@ -230,6 +230,14 @@ public static function getTab($idLang, $idTab)
 					ON (t.`id_tab` = tl.`id_tab` AND tl.`id_lang` = '.(int) $idLang.')
 				WHERE t.`id_tab` = '.(int) $idTab.(defined('_PS_HOST_MODE_') ? ' AND `hide_host_mode` = 0' : '')
             );
+
+            $lang = new Language($idLang);
+            $tabLang = new TabLang(Context::getContext()->getTranslator(), $lang->locale);
+            $newName = $tabLang->getName($row['class_name']);
```
Comment (thierrymarianne) [inline classes/Tab.php]: Where does `row` come from?

**Comment by aleeks** [comment]
Hello @julienbourdeau,
1/ can we import in crowdin all translation present in data file ?
2/ *can you update your licence with new ?!!! *
3/ update your phpdtorm licences 😎

By the way, very good idea

File: classes/Tab.php (hunk 1 above)
Comment (julienbourdeau) [inline classes/Tab.php]: Oops!

**Comment by aleeks** [comment]
**See with @aleeks and @AlexEven before merge (localisation file)! **

**Comment by aleeks** [comment]
@julienbourdeau could you rebase please ? hihi

**Comment by aleeks** [comment]
Closing this PR @julienbourdeau because of  https://github.com/PrestaShop/PrestaShop/pull/7363 ?

**Pull request closed without merging**